python/cudf/cudf/core/column/lists.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import pyarrow as pa
+
+import cudf
+from cudf.core.column import ColumnBase
+from cudf.core.column.methods import ColumnMethodsMixin
+from cudf.core.dtypes import ListDtype
+from cudf.utils.dtypes import is_list_dtype
+from cudf.utils.utils import buffers_from_pyarrow
+
+
+class ListColumn(ColumnBase):
+    def __init__(
+        self,
+        data,
+        size,
+        dtype,
+        mask=None,
+        offset=0,
+        null_count=None,
+        children=(),
+    ):
+        super().__init__(
+            data,
+            size,
+            dtype,
+            mask=mask,
+            offset=offset,
+            null_count=null_count,
+            children=children,
+        )
+
+    @property
+    def base_size(self):
+        return self.size
+
+    @property
+    def values(self):
+        """
+        Column of values (may itself be a ListColumn)
+        """
+        return self.children[0]
```

Comment:
`values` is somewhat ambiguous because of Pandas `.values`. Should we rename this property to something else even though PyArrow defines it this way?

Reply:
How about `elements`?

Reply:
We should probably use `elements` consistently in `ListColumn`/`ListDtype` if we decide to do this

Reply:
Elements is still ambiguous in my mind. Can we consistently use `leaves`?

Reply:
But the `values` of list column of dtype `ListDtype(ListDtype(ListDtype('int64')))` are of type `ListDtype(ListdType('int64'))` which is different from the leaves

Reply:
Apologies, had a brainfart and misunderstood this API. `elements` sounds good to me then